### DISCUSSION TITLE
Project Creation Completed The ProjectResource ApplicationPageBackgroundThemeBrush prompt was not found

### DISCUSSION THREAD

**Discussion by hui3215** [unoplatform/uno#9403] [Q&A]
Type "ThemeResource" was not found. Make sure that assembly references are not missing and that all referenced assemblies have been generated

**Comment by jeromelaban** [comment]
Thanks for the report. This is a known visual studio issue that [you can work around using this](https://platform.uno/docs/articles/get-started-wizard.html#the-xaml-editor-shows-the-type-page-does-not-support-direct-content-message).

**Comment by hui3215** [comment]
I built all the projects in the way provided on the site and reopened the solution, and then the editor still prompted for an error, and then when using bindings, I used {x:Bind xxxx} to report an error, but I could use {Binding xxxx}, and the editor did not do any code hints
